reference/carts.sf.yml

```diff
@@ -250,6 +250,45 @@ paths:
       responses:
         '200':
           $ref: '#/components/responses/deleteCartsItems'
+  '/cart/{cartId}/currency':
```

Comment:
@bc-mackxu This seemed to work with both `cart` and `carts`. Can we use `carts` here so it is consistent with the other endpoints? Results for `carts` pictured
 
![image](https://user-images.githubusercontent.com/29524703/161861544-b6820b54-7bb0-4d2f-8de7-0a5fe1320c1c.png)

Reply:
Thank you @markcmurphy, yes, please update and consistent with other endpoints.

Reply:
@traci707 FYI ^^